spec/status.xml

```diff
@@ -5,62 +5,47 @@
-its predecessor is a change of the default treatment of the <att>ttp:markerMode</att> parameter attribute
-from <code>continuous</code> to <code>discontinuous</code>. A cumulative summary of all changes applied
-to this version since the Recommendation was first published is available at
-<loc  href="ttml1-changes.html#change-history-rec-to-2e-per">Changes
-from Recommendation to Proposed Edited Recommendation (2nd Ed.)</loc> and
-<loc  href="ttml1-changes.html#change-history-2e-per-to-rec-2e">Changes
-from Proposed Edited Recommendation (2nd Ed.) to Recommendation (2nd Ed.)</loc>.</p>
+<p>Substantive changes applied since the <loc href="&prevloc;">previous version of this specification</loc> are listed at 
+<loc href="substantive-changes-summary.txt">substantive-changes-summary.txt</loc>.</p>
 
-<p>A <loc href="http://www.w3.org/2008/10/dfxp-testsuite.zip">sample test suite</loc> is available. This set of tests is not intended to
-be exhaustive. The <loc href="http://www.w3.org/2009/05/dfxp-results.html">implementation report</loc> used by the director to
-transition to Recommendation has been made available. There have been no formal objections
-to the publication of this document.</p>
+<p>For convenience, a diff between this and the <loc href="&prevrec;">previous edition</loc> 
+is offered at the <loc href="https://services.w3.org/htmldiff?doc1=https%3A%2F%2Fwww.w3.org%2FTR%2F2013%2FREC-ttml1-20130924%2F%26doc2=https%3A%2F%2Fwww.w3.org%2FTR%2F2018%2FCR-ttml1-20180313%2F">W3C HTML Diff service</loc>.</p>
 
-<p>This document was produced by the <loc
-href="http://www.w3.org/AudioVideo/TT/">Timed Text (TT) Working Group</loc> as
-part of the W3C <loc href="http://www.w3.org/2008/WebVideo/">Video
-in the Web Activity</loc>, following the procedures set out for the W3C <loc
-href="http://www.w3.org/Consortium/Process/">Process</loc>.  The authors of this
-document are listed in the header of this document, including input and review
-by liaisons from the <loc href="http://www3.ebu.ch/cms/en/home">European
-Broadcasting Union (EBU)</loc>, and the <loc href="https://www.smpte.org">Society
-of Motion Picture &amp; Television Engineers (SMPTE)</loc>.</p>
+<p>For this specification to exit the CR stage, at least 2 independent implementations of every <emph>CR exit criteria test</emph> is required, as documented in the Working Group's <loc href="">implementation report</loc>. The Working Group does not require that implementations are publicly available but encourages them to be so.</p>
 
-<p>If you wish to report errors in or submit comments on this document, then please
-submit them to <loc href="mailto:public-tt@w3.org">public-tt@w3.org</loc>
-(<loc href="mailto:public-tt-request@w3.org">subscribe</loc>,
-<loc href="http://lists.w3.org/Archives/Public/public-tt/">archive</loc>). All comments are welcome.</p>
+<p>The Working Group has not identified features "at risk" for this specification.</p>
```

Comment:
I don't think we need to say this - or is it a pubrules requirement?

Reply:
> § It must identify any "features at risk" declared by the Working Group (as defined in section 7.4 of the W3C Process Document).

https://www.w3.org/pubrules/doc/rules/?profile=CR

Reply:
Right, but no features are at risk here. It's not harmful to state that, I suppose.

Reply:
My thought exactly. I felt like experimenting with pubrules as little as possible and have copied text that we used in the past with IMSC in similar circumstances.